docs/source/usage.rst

```diff
@@ -172,3 +172,55 @@ the XML URL from Addons Server in ``remote``:
       "local": "https://kinto.stage.mozaws.net/v1/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/",
       "remote": "https://blocklist.addons.mozilla.org/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/"
     }
+
+How to create the ``blockpages-generator`` function on Amazon Lambda?
+---------------------------------------------------------------------
+
+1. `Log into the Amazon Web Service console. <https://console.aws.amazon.com/lambda/home>`_
+2. Click the **Create a Lambda Function** button
+3. Skip the blueprint
+4. Configure the function: **name**, **description**
+5. Select **Python 2.7** runtime
+6. Select **Upload a file from Amazon S3**
+7. Enter your S3 ``lambda.zip`` URL
+8. Use **lambda.blockpages_generator** handler
+9. Set a **Lambda S3 Exec Role**
+10. Configure a 5 minutes timeout
+11. You can proceed without a VPC
+
+.. image:: images/blockpages-generator-setup.png
+    :align: center
+
+
+How to run the ``blockpages-generator`` function on Amazon Lambda?
+------------------------------------------------------------------
+
+Once your function has been configured, you can run it.
+
+To do that, select the function and in the **Actions** menu select
+**Configure Test event**.
+
+You will then be able to enter a **Input test event** as a JSON.
+
+.. image:: images/blockpages-generator-test-event.png
+    :align: center
+
+The ``blockpages-generator`` let you define the following inputs:
+
+- **server**: The Kinto server to read the blocklists data from.
+- **bucket**: The Kinto bucket the blocklist are in. default: *blocklists*
```

Comment:
Sorry, I see that you did already mention that `bucket` refers to a Kinto bucket. Maybe emphasize `bucket` is where the blocklists are read from and not the S3 bucket? This is going to be hard because the parameters aren't well-named.